client_log.go

```diff
@@ -27,16 +26,10 @@ type LogUploader interface {
 	Upload(api ApiRequester, server string, logs LogData) error
 }
 
-type LogEntry struct {
-	Timestamp string `json:"timestamp"`
-	Level     string `json:"level"`
-	Message   string `json:"message"`
-}
-
 type LogData struct {
 	deviceID     string
 	deploymentID string
-	Messages     []LogEntry `json:"messages"`
+	Messages     []byte `json:"messages"`
```

Comment:
As agreed logs are written as a JSON entries to the log file. There is no need to marshal those and convert to `LogEntry` struct as the raw bytes returned from file are already JSON.